src/current/_includes/common/numa_and_go.md

```diff
@@ -0,0 +1,8 @@
+In a [NUMA (non-uniform memory access) architecture](https://en.wikipedia.org/wiki/Non-uniform_memory_access), the system’s memory is physically distributed across multiple memory banks or "nodes", and each node is assigned to a processor. A processor can access its local memory much faster than non-local memory. This non-uniform memory access can lead to performance differences depending on data location.
+
+CockroachDB is written in Go, which has no support for NUMA scheduling or pinning. To run multiple CockroachDB clusters on a NUMA architecture:
```

Comment:
> which has no support for NUMA scheduling or planning

change to:

> which has no native support for NUMA scheduling or planning

Small nit: Go can't constrain itself to a NUMA node, but the OS can.